include/multipass/standard_paths.h

```diff
@@ -45,7 +45,7 @@ class StandardPaths : public Singleton<StandardPaths>
     static constexpr auto PicturesLocation = StandardLocation::PicturesLocation;
     static constexpr auto TempLocation = StandardLocation::TempLocation;
     static constexpr auto HomeLocation = StandardLocation::HomeLocation;
-    static constexpr auto DataLocation = StandardLocation::DataLocation;
+    static constexpr auto DataLocation = StandardLocation::AppDataLocation;
```

Comment:
I understand this saves us from changing client code everywhere, but I think we should stick to what we're wrapping. These wrappers exist only for the purpose of testing and I believe we should keep them as thin and faithful to the original as possible. The moment we start mapping things differently or adding functionality, they themselves would need to be tested.

Reply:
I see your point. Now that I look closer, `DataLocation` is not used anywhere in our code and the `QStandardPaths::StandardLocation::DataLocation` has been removed in favor of `QStandardPaths::StandardLocation::AppDataLocation` and we already have a `AppDataLocation` constant that wraps that value. So I guess the `DataLocation` can be removed entirely, right?

Reply:
Oh ok, yep I think it should be removed if it is not available in the *wrappee* anymore.